Remotion/Web/Development.WebTesting.IntegrationTests/BrowserLogEntryTest.cs

```diff
@@ -41,19 +41,28 @@ public void BrowserLogEntry_ShouldWrapSeleniumLogEntry ()
       var js = JavaScriptExecutor.GetJavaScriptExecutor(home.Context.Browser);
       js.ExecuteScript($"console.error('{errorMessage}')");
 
-      var browserLogEntries = ((IWebDriver)home.Context.Browser.Driver.Native)
-          .Manage().Logs.GetLog(LogType.Browser);
+      IReadOnlyCollection<BrowserLogEntry> browserLogEntries = null;
+      var logger = Helper.LoggerFactory.CreateLogger("BrowserLogEntryTest");
+      RetryUntilTimeout.Run(
+          logger,
+          () =>
+          {
+            browserLogEntries = home.Context.Browser.GetBrowserLogs();
+            if(browserLogEntries.Count == 0)
+              throw new Exception("No browser logs found");
+          });
 
-      var errorLogEntry = browserLogEntries.Single(log => log.Message.Contains(errorMessage));
-      var wrappedErrorLogEntry = new BrowserLogEntry(errorLogEntry);
+      var wrappedErrorLogEntry = browserLogEntries.Single(log => log.Message.Contains(errorMessage));
 
       Assert.That(wrappedErrorLogEntry.Level, Is.EqualTo(LogLevel.Severe));
       Assert.That(
           wrappedErrorLogEntry.Timestamp,
           Is.InRange(DateTime.UtcNow.Subtract(maxExpectedTestRunTime), DateTime.UtcNow.Add(maxExpectedTestRunTime)));
-      Assert.That(wrappedErrorLogEntry.Message, Does.Match($@"^console-api \d+:\d+ ""{errorMessage}""$"));
+      Assert.That(wrappedErrorLogEntry.Message, useBiDiLogEntry ? Is.EqualTo(errorMessage) : Does.Match($"""^console-api \d+:\d+ "{errorMessage}"$"""));
 
-      var logAsStringPattern = $@"^\[\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\dZ\] \[Severe\] console-api \d+:\d+ ""{errorMessage}""$";
+      var logAsStringPattern = useBiDiLogEntry
+          ? $@"^\[\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\dZ\] \[Severe\] {errorMessage}$"
+          : $@"^\[\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\dZ\] \[Severe\] console-api \d+:\d+ ""{errorMessage}""$";
```

Comment:
Since you reformatted the string interpolation in line 61 but kept it in lines 64 and 65 I would recommend reverting the syntax change in line 61 or updating all lines to match.